docs/UserGuide.adoc

```diff
@@ -389,13 +389,17 @@ Clears all entries from the exam book. All exams will be cleared in the address
 Format: `clearexams`
 //end::Exam[]
 
+//tag::Fees[]
+[[Fees]]
 === Fees Commands
 Below are the commands that deals with fees data:
 
-==== Adding fees to a person: `addfees` image:Admin.png[16,16]
+==== Editing fees of a person: `editfees` image:Admin.png[16,16]
 
-Adds a numerical fee to a person in the address book. +
-Format: `add INDEX FEE`
+Edits the numerical fees of a person in the address book. +
+Format: `edit INDEX FEE DUE_DATE` +
```

Comment:
The format here should be 'editfees INDEX FEE DUE_DATE' Not just edit ....

Reply:
Thanks Meg!!!